pyproject.toml

```diff
@@ -34,7 +34,7 @@ classifiers = [
 requires-python = ">=3.8"
 
 dependencies = [
-    "hapi-schema== 0.7.4",
+    "hapi-schema@git+https://git@github.com/OCHA-DAP/hapi-sqlalchemy-schema@develop#egg=hapi-schema",
```

Comment:
I think there is a release of the hapi-schema that has the refugees table - https://github.com/OCHA-DAP/hapi-sqlalchemy-schema/blob/main/changelog.md#080

Reply:
Thanks!  I'll fix this and some other conflicts and then merge.